sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormPage.cs

```diff
@@ -47,20 +48,22 @@ internal FormPage(PageResult_internal pageResult, IReadOnlyList<ReadResult_inter
         public float Height { get; set; }
 
         /// <summary>
-        /// The unit used by the width, height and boundingBox properties. For images, the unit is &quot;pixel&quot;.
-        /// For PDF, the unit is &quot;inch&quot;.
+        /// The unit used by the width, height and <see cref="BoundingBox"/> properties. For images, the unit is
+        /// &quot;pixel&quot;. For PDF, the unit is &quot;inch&quot;.
         /// </summary>
         public LengthUnit Unit { get; set; }
 
         /// <summary>
-        /// When includeTextDetails is set to true, a list of recognized text lines. The maximum number of lines returned
-        /// is 300 per page. The lines are sorted top to bottom, left to right, although in certain cases proximity is
-        /// treated with higher priority. As the sorting order depends on the detected text, it may change across images
-        /// and OCR version updates. Thus, business logic should be built upon the actual line location instead of order.
+        /// When <see cref="RecognizeOptions.IncludeTextContent"/> is set to <c>true</c>, a list of recognized text lines.
```

Comment:
It's also always populated if you call RecognizeContent.

Reply:
There's a `For calls to recognize content, this list is always populated.` below. Do you think we should rephrase it?

Reply:
ah, no.  Missed that.  Thanks!